tests/util/test_defn.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from src.util.defn import JobSettings
+from pydantic import ValidationError
+
+def test_job_settings_creation():
+    job_settings_data = {
```

Comment:
One thing that made sense from Suzanne was to add docstrings to the test methods, just to make sure it's clear for all levels of users/developers and when we revisit this test years later. I recommend adding if you have time. Thanks!

Reply:
I am going to put that in the "Docstring" PR.